source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,481 @@
+VM, highlight the `clear-linux-project:clear-linux-os:basic:` string and copy
+it to your clipboard. For the version you can use the label `latest` instead
+of referencing a specific version, which is what we will do when we create our
+VM in a moment.
+
+Create a MS Azure resource group
+********************************
+
+With all the information gathered, we need to create a resource group to
+manage multiple resources within MS Azure for our |CL| VM. To learn more about
+resource groups, visit the `Azure Resource Manager overview`_ for an overview
+and detailed description of resources within MS Azure.
+
+To create our new resource group, run the :command:`az` command shown below to
+create a resource group named `ClearResourceGroup` and locate it in the
+`westus` region.
+
+.. code-block:: console
+
+   az group create -n ClearResourceGroup -l westus
+
+When the command has completed, the output from this command is similar to
+the following:
+
+.. code-block:: console
+
+   {
+     "id": "/subscriptions/{unique-id}/resourceGroups/ClearResourceGroup",
+     "location": "westus",
+     "managedBy": null,
+     "name": "ClearResourceGroup",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
+Create a |CL| virtual machine
+*****************************
+
+To create a new |CL| VM, run the following :command:`az` command using the
+URN `:clear-linux-project:clear-linux-os:basic:latest` that we located earlier
+in our search for the |CL| images available in the MS Azure marketplace:
+
+.. code-block:: console
+
+   az vm create --resource-group ClearResourceGroup --name ClearVM --image clear-linux-project:clear-linux-os:basic:latest --generate-ssh-keys
+
+.. note::
+
+   If you have already defined your public/private SSH key pair and they are
+   stored in your :file:`$HOME/.ssh` directory, you do not need to include the
+   :parameter:`--generate-ssh-keys` parameter.
+
+Your output from this command will look similar to this output, where $USER is
+your user name:
+
+.. code-block:: console
+
+   SSH key files '/home/$USER/.ssh/id_rsa' and '/home/$USER/.ssh/id_rsa.pub' have been generated under ~/.ssh to allow SSH access to the VM. If using machines without permanent storage, back up your keys to a safe location.
+
+   running...
+
+   {
+     "fqdns": "",
+     "id": "/subscriptions/{unique-id}/resourceGroups/ClearResourceGroup/providers/Microsoft.Compute/virtualMachines/ClearVM",
+     "location": "westus",
+     "macAddress": "00-0D-3A-37-C7-59",
+     "powerState": "VM running",
+     "privateIpAddress": "10.0.0.4",
+     "publicIpAddress": "13.91.4.245",
+     "resourceGroup": "ClearResourceGroup",
+     "zones": ""
+   }
+
+Take note of the public IP address from your output. To login into the new
+|CL| VM, run the :command:`ssh` command with the public IP address listed as
+shown:
+
+.. code-block:: console
+
+   ssh 13.91.4.245
```

Comment:
> +   ssh 13.91.4.245

Do you not need to add the username in front of the IP address like: ssh user@13.91.4.245?